docs/rfcs/1020-pki-support.md

```diff
@@ -0,0 +1,291 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+# PKI Support
+
+## Overview
+
+Add support to use Parsec with an externally managed PKI
+
+## Background & Motivation
+
+Some organizations manage their own PKI, and want Parsec to rely on that PKI.
+
+That custom PKI come in the form of certificate with their private key often loaded in a dedicated device (a smartcard/token like a YubiKey).
+They would like that we use their certificate to access Parsec, so the user does not have to use a password or rely on the keyring of the OS.
+
+There is also an opportunity to rely on that PKI to join an organization.
+
+Put in a simpler term access Parsec using a Smartcard.
+
+### Terms:
+
+- **PKI**: Public Key Infrastructure
+- **Smartcard** or **Token**:
```

Comment:
I'd rather avoid using `Token` here since we already use token in multiple places in the code base for something different